content/registration/adopt-a-ticket.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Adopt-a-ticket"
+page_header_bg: "images/background/page-title-bg.jpg"
+date: 2024-02-10T10:40:24+01:00
+description: "Adopt-a-ticket is your chance to support FOSS4GE 2025 by
+purchasing any number extra tickets for the conference that will be donated to
+our wonderful workshop instructors and Travel Grant Program (TGP) receivers."
+draft: false
+---
+
+Adopt-a-ticket is your chance to support FOSS4GE 2025 by purchasing any number of
+extra tickets for the conference that will be donated to our wonderful workshop
+instructors and Travel Grant Program (TGP) receivers.
+
+## Why?
+- We are dedicated to running the
+[**Travel Grant Program**](../../registration/travel-grant/) (TGP) - a way to
+facilitate accessibility and diversity in the event. In addition to a
+monetary grant for grantees' travel and accommodation expenses handled in
+cooperation with OSGeo, the LOC will grant free access to the conference main
+days plus one workshop of choice for all Travel Grant receivers.
+- the LOC understands that preparing and conducting an educative and
+successful workshop takes quite some effort and time. This is the reason we are
+giving away free tickets to all workshop instructors.
+
+We do this because we believe this is the right thing to do.
+
+But this also means that the conference will miss out on some of the income funds
+although we still need to cover the related expenses (lunches, coffee breaks, etc).
+This is where we would appreciate your help. Instead of increasing the ticket
+cost for all we chose to allow you to make a conscious choice.
+
+## How does this work?
+When registering for the conference, you can choose between any of the add-ons: a
+t-shirt, a mug, workshop tickets and so on. One of the add-ons there is
+called _Adopt-a-ticket_, your chance to help us raise extra funds to do the
+right thing.
+
+We see it equally as an opportunity for smaller organizations and individuals
+who would like to support the conference but for whom the existing
+[**sponsorship levels**](../../call-for-sponsors/) are too expensive.
+
+## What's in it for you?
+With _Adopt-a-ticket_ you can purchase any number of extra tickets. In return
+we would like to show our gratitude by having you included in the
+[**Honorable mentions**](../../sponsors/honorable-mentions/) section of the

```

Comment:
change ref to 

```
[**Honorable mentions**](/sponsors/honorable-mentions/)
```

plus add sponsors/honorable-mentions.md file, can be empty but otherwise the link is broken (HTTP 404)